tick/stateful/expr_test.go

```diff
@@ -1321,10 +1334,10 @@ func runCompiledEvalTests(
 	}
 }
 
-func mustCompileExpression(t *testing.T, node tick.Node) stateful.Expression {
+func mustCompileExpression(node tick.Node) stateful.Expression {
 	se, err := stateful.NewExpression(node)
 	if err != nil {
-		t.Fatalf("Failed to compile expression: %v", err)
+		panic(fmt.Sprintf("Failed to compile expression: %v", err))
```

Comment:
Why we do panic here? If one test is failing, we fail here the whole test suite.
By using "Fatal" we fail this test only.